Vizzuality/heco-invest
# Add sections to public project page

## Description
This PR adds the following sections to the public page of the project:

- Overview
- Impact
- Funding & Development
- Project developers
- Contact

![Screenshot 2022-05-25 at 19 49 22](https://user-images.githubusercontent.com/51995866/170330321-53351f91-955d-4ec6-b99a-32942a75cfae.png)
![Screenshot 2022-05-25 at 19 51 29](https://user-images.githubusercontent.com/51995866/170330327-46b30591-67b9-4b84-aafc-abe03009557c.png)

**Suggestion for another PR**: 
- Impact Modal should be used in the About section, that's where it's taken from.
- Adapt the margins of the main footer. It is not aligned with the rest of the sections.
- Add the contact modal, which is in the [this PR](https://github.com/Vizzuality/heco-invest/pull/209).
- Add municipality layer to map.

## Testing instructions
You can enter the following example created using this url and check if the components are displayed correctly:
`/project/test-10-test-30`

## Tracking
[LET-457](https://vizzuality.atlassian.net/browse/LET-457)
[LET-458](https://vizzuality.atlassian.net/browse/LET-458)
[LET-459](https://vizzuality.atlassian.net/browse/LET-459)
[LET-508](https://vizzuality.atlassian.net/browse/LET-508)
[LET-509](https://vizzuality.atlassian.net/browse/LET-509)
